.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello and thanks for contributing!
+
+        Before continuing, can you check if the answer is available within:
+
+        * Guides and examples - https://www.mapbox.com/ios-sdk/navigation/
+        * API reference - https://www.mapbox.com/mapbox-navigation-ios/navigation/
+        * The example app - https://github.com/mapbox/navigation-ios-examples
```

Comment:
Should we mention mapbox/mapbox-maps-ios and mapbox/mapbox-directions-swift as other likely places to file issues in?

Reply:
I like that

Reply:
done